Applying kore hardfork change (#518)
klaytn/klaytn-docs: docs/klaytn/design/computation/computation-cost/computation-cost.md

The below table shows the computation cost of EVM opcodes. The computation cost was determined based on experiments.

{% hint style="success" %}
NOTE: Computation costs have changed with the `Kore` hardfork. If you want the previous document, please refer to [previous document](computation-cost-previous.md).

`Kore` hardfork block numbers are as follows.

> * Baobab Testnet: please refer [this](https://github.com/klaytn/klaytn/blob/b01f8f3b7aa69afc1f7693e652adc5f2d8fa4ab6/params/config.go#L79)

Reviewer: ```suggestion
* Baobab Testnet: please refer to this [link](https://github.com/klaytn/klaytn/blob/b01f8f3b7aa69afc1f7693e652adc5f2d8fa4ab6/params/config.go#L79)
```

Author: reflected at https://github.com/klaytn/klaytn-docs/pull/518/commits/a11e18981c19918f07a0639d51894aaf45987c61

> * Cypress Mainnet: please refer [this](https://github.com/klaytn/klaytn/blob/b01f8f3b7aa69afc1f7693e652adc5f2d8fa4ab6/params/config.go#L49)

Reviewer: ```suggestion
* Cypress Mainnet: please refer to this [link](https://github.com/klaytn/klaytn/blob/b01f8f3b7aa69afc1f7693e652adc5f2d8fa4ab6/params/config.go#L49)
```

Author: reflected at https://github.com/klaytn/klaytn-docs/commit/a11e18981c19918f07a0639d51894aaf45987c61

> * Cypress Mainnet: please refer [this](https://github.com/klaytn/klaytn/blob/b01f8f3b7aa69afc1f7693e652adc5f2d8fa4ab6/params/config.go#L49)

Reviewer: ```suggestion
* Cypress Mainnet: please refer to this [link](https://github.com/klaytn/klaytn/blob/b01f8f3b7aa69afc1f7693e652adc5f2d8fa4ab6/params/config.go#L49)
```

Author: reflected at https://github.com/klaytn/klaytn-docs/commit/a11e18981c19918f07a0639d51894aaf45987c61

> * Cypress Mainnet: please refer [this](https://github.com/klaytn/klaytn/blob/b01f8f3b7aa69afc1f7693e652adc5f2d8fa4ab6/params/config.go#L49)

Reviewer: ```suggestion
* Cypress Mainnet: please refer to this [link](https://github.com/klaytn/klaytn/blob/b01f8f3b7aa69afc1f7693e652adc5f2d8fa4ab6/params/config.go#L49)
```

Author: reflected at https://github.com/klaytn/klaytn-docs/commit/a11e18981c19918f07a0639d51894aaf45987c61

> * Cypress Mainnet: please refer [this](https://github.com/klaytn/klaytn/blob/b01f8f3b7aa69afc1f7693e652adc5f2d8fa4ab6/params/config.go#L49)

Reviewer: ```suggestion
* Cypress Mainnet: please refer to this [link](https://github.com/klaytn/klaytn/blob/b01f8f3b7aa69afc1f7693e652adc5f2d8fa4ab6/params/config.go#L49)
```

Author: reflected at https://github.com/klaytn/klaytn-docs/commit/a11e18981c19918f07a0639d51894aaf45987c61

The below table shows the computation cost of EVM opcodes. The computation cost was determined based on experiments.

{% hint style="success" %}

> NOTE: This document is written based on the latest EVM which have been used since the `Kore` hardfork.

Reviewer 2: It's hard to understand which one you want to explain after Kore. If this section explains about computation cost basis of the latest EVM opcodes, adding the target object seems good. 
```suggestion
NOTE: This document(chapter or section) explains(or discusses) about computation cost basis of the latest EVM opcodes which have been used since the `Kore` hardfork.
```

Author: Good point. What about this? Reading it again, it seems that there is no need to talk about EVM..
```
NOTE: Computation costs have changed with the `Kore` hardfork.
```

Author: changed at https://github.com/klaytn/klaytn-docs/pull/518/commits/06f6a658ce86ba316298436e84c23393a0855b03

If you want the previous document, please refer to [previous document](computation-cost-previous.md).

> `Kore` hardfork block number is as follows.

Reviewer 2: ```suggestion
`Kore` hardfork block numbers are as follows.
```

The below table shows the computation cost of EVM opcodes. The computation cost was determined based on experiments.

{% hint style="success" %}

> NOTE: Computation costs have changed with the `Kore` hardfork. If you want the previous document, please refer to [previous document](computation-cost-previous.md).

Reviewer 3: Don't we also need to update computation-cost-previous.md?

Author: the previous page contains the initial vm. 

However, as mentioned in https://github.com/klaytn/klaytn-docs/pull/518#discussion_r1174840828, i'm going to replace those pages to a one summary page or add the descriptions about the changes at each page later.
